_posts/2020-12-01-Fortran-Newsletter-December-2020.md

```diff
@@ -0,0 +1,191 @@
+* [llvm mirror PR#87: Enable github actions for llvm](https://github.com/flang-compiler/llvm/pull/87)
+* [flang-driver PR#94: Enable github actions](https://github.com/flang-compiler/flang-driver/pull/94)
+
+The Classic Flang biweekly call has been set up to discuss issues and plans
+for the next pull requests to be validated and merged. Our next calls are Wednesday, November 16 and 30, 8:00 AM Pacific time. The notes from previous calls, upcoming agenda and a link to join the call can be found
+[here](https://docs.google.com/document/d/1-OuiKx4d7O6eLEJDBDKSRnSiUO2rgRR-c2Ga4AkrzOI).
+
+### LLVM Flang
+
+Work continues on LLVM Flang, concentrating on semantics, lowering and runtime. Significant contributions are being made for OpenMP and OpenACC support.
+
+In conjunction with the MLIR-based code from the _fir-dev_ fork (the Fortran IR used for lowering), Flang can compile and run most F77 programs, including the Fortran Compiler Validation Suite (FCVS).
+
+Pat McCormick is (still) working on an RFC for the merge of the lowering code
+in the fir-dev fork into LLVM master. (This was interrupted by Supercomputing 2020 and other ECP duties.) The goal is to expedite this in a way that is acceptable to the Flang community, so we can do further work in the single master branch.
+
+Recent updates include:
+* Johannes Doerfert has created a web page at https://flang.llvm.org; you can find call and Slack logistics there
+* Nichols Romero has an llvm-dev RFC for adding Fortran tests to the llvm-tests project: http://lists.llvm.org/pipermail/llvm-dev/2020-November/146873.html
+* Andzrej Warzynski has a flang-dev RFC regarding flang option names: http://lists.llvm.org/pipermail/flang-dev/2020-November/000588.html
+* Andzrej Warzynski has a cfe-dev RFC regarding refactoring clang to help flang driver become independent of clang: http://lists.llvm.org/pipermail/cfe-dev/2020-November/067263.html
+* Changed representation of CHARACTER data in type system to make more consistent with other types (for arrays)
+* Changed COMPLEX expression representation to provide better handling in lowering
+* More improvements for supporting Fortran 77 programs
+* Implemented runtime support for basic ALLOCATE/DEALLOCATE and further work
+* Continued implementation of table-driven runtime for derived types; posted documentation
+* Continued implementation of array expression lowering
+* Improved error checks on forward references
+* More updates to flang driver (option handling; -E can now be used to invoke just the Flang preprocessor)
+* OpenACC semantic checks for modifiers on enter/exit data, set directives
+* OpenACC lowering (enter/exit data, update, init, shutdown, wait directives)
+* OpenMP structure checker updates; semantic checks for copyin clause; schedule class
+
+### LFortran
+
+What's new in LFortran:
+
+TODO @certik
+
+You can follow LFortran on Twitter for latest updates: [@lfortranorg](https://twitter.com/lfortranorg).
```

Comment:
I don't have many updates for LFortran for November, I have been super busy with other things. There will be some good updates for December though.

```suggestion

```